src/components/copy/index.tsx

```diff
@@ -21,6 +21,8 @@ export enum CopySize {
 const StyledCopy = styled.p`
 	margin: 0;
 	line-height: 1.5;
+	user-select: none;
```

Comment:
:/ Sure?

Reply:
I tried to imitate the native behaviour of an native App with that... Do you think it’s not a good idea?

Reply:
Its the discussion from #519 again, si? 

Personally I am annoyed when apps prevent me from copying things. E.g. I might want to explain what part of the UI I am writing about by citing the text I found there. Being able to copy such text makes my life as a user considerably easier then.

Reply:
That being said there are legit use cases for `user-select: none`, e.g. draggable ui elements